virttest/qemu_devices/qcontainer.py

```diff
@@ -1969,23 +1985,26 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
             else:       # very old qemu without 'addr' support
                 devices.append(qdevices.QOldDrive(name, use_device))
 
+            format_node = devices[-1]
```

Comment:
1. what about  keeping the original code for '-drive', I know in the following code sections(line 2148), you use format_node instead of device[-1], but that code section looks like used for -drive only, so we just keep it the same as before

Also, I see that in some code section used for '-drive', you didn't do the code change (change device[-1] to format_node), so I suggest we don't do code change for '-drive' code sections.

2. I am afraid devices[-2] in line 2136 and line 2138 also needs to be replaced

Reply:
1.If it does distinguish the blockdev and drive , keep device[-1] just for keep impact scope is small. 
The format_node for blockdev mode or it does not  distinguish the blockdev and drive. so i think it keep this have better consistency.
2. will update.